pkgs/development/dotnet-modules/python-language-server/format-deps.sh

```diff
@@ -0,0 +1,34 @@
+# Retrieve sha256 hashes for each dependency in and format fetchNuGet calls
+echo "" > deps.nix
+urlbase="https://www.nuget.org/api/v2/package"
+cat << EOL
```

Comment:
It would be nice if there was some sort of disclaimer at the top of the file that this is autogenerated, and what script should be run to (re)generate it.

Reply:
Done